app/src/components/common/TmSessionImportName.vue

```diff
@@ -74,6 +87,14 @@ const nameExists = (value) => {
   }
 }
 
+const cryptoDictionary = {
```

Comment:
should we put the visible name also in the API to not maintain the data in 2 places?

Reply:
ping

Reply:
> should we put the visible name also in the API to not maintain the data in 2 places?

You mean in the database right? Sure, we can do that

Reply:
so I could just include this very same dictionary as a JSON in the database right. This is what you mean?

Reply:
This is what I was thinking of https://github.com/luniehq/lunie/pull/4481/files#r458440772
Not sure if we want to do this here. But if we do it later we have again a breaking change.